### ISSUE TITLE
Customizing ChannelListVC

### ISSUE THREAD

**Issue by kcollignon** [GetStream/stream-chat-swift#1077]
Hi there,

I was wondering if it's possible to easily customize the ChannelListVC to simply remove the "create new channel" button at the top, or if this would involve creating our own custom view?

Thanks,

**Comment by VojtaStavik** [comment]
Hi @kcollignon, we're just days away from releasing a new version of the SDK which will allow full customizability of all UI components. You will be able to do something like this:
```swift
class MyController: ChannelListVC { 
  override func viewDidLoad() { 
    super.viewDidLoad()
    // remove the "new channel button"
    navigationItem.rightBarButtonItem = nil
  }
}
```
 Please stay tuned, it should happen within next week 👍

**Comment by b-onc** [comment]
Hello @kcollignon 

We've released 4.0-beta.1 which has the customization API totally open. Please check it out!

**Comment by kcollignon** [comment]
Hi @b-onc thank you so much for the update! I will follow the 4.0 beta and I look forward to customizing! Quick question, with the new subclass options is it possible to override the data sources for some of the components? For example, the default "Create new Group" UI that is in the DemoApp, is it possible to swap the data source for the list of users to pull from my own API that would have some extra user permissions baked in? Thanks again.
